Source/OnlineSubsystemEpic/Classes/EpicOnlineServicesLoginTask.cpp

```diff
@@ -5,6 +5,7 @@
 #include "OnlineSubsystemUtils.h"
 #include "OnlineSubsystemEpicTypes.h"
 #include "Utilities.h"
+#include "Private/Utilities.h"
```

Comment:
Please check if you and your team have configured the build process correctly. You shouldn't need the *Private* part of the include path.